_partials/_selfhosted_cta.md

```diff
@@ -0,0 +1,4 @@
+<Highlight type="cloud" header="Want to skip these steps?" button="Try for free">
+Deploy a Timescale service in the cloud. Don't worry about performance, scalability, high availability, backups or management.
```

Comment:
... in the cloud. We tune your service for performance ... so you can sit back and relax.